src/lib/agent/Agent.ts

```diff
@@ -37,6 +37,10 @@ import { ConnectionsModule } from '../modules/ConnectionsModule';
 import { RoutingModule } from '../modules/RoutingModule';
 import { BasicMessagesModule } from '../modules/BasicMessagesModule';
 import { LedgerModule } from '../modules/LedgerModule';
+import { CredentialsModule } from '../modules/CredentialsModule';
+import { CredentialService } from '../protocols/credentials/CredentialService';
```

Comment:
Any reason for `CredentialService` vs `CredentialsModule` (singular vs plural)? I see connection has the same difference. It makes sense in a way.

Reply:
Hmm, you got me :) `CredentialService` similarly like `CredentialRepository` is about the work with one entity. Plural naming in `CredentialsModule` is about the work with various parts related to credentials, but it's perhaps driven more by outside API.